generators/page/templates/vue/src/main/webapp/app/pages/page.component.ts.ejs

```diff
@@ -0,0 +1,18 @@
+import { mixins } from 'vue-class-component';
+
+import { Component, Inject } from 'vue-property-decorator';
+import Vue2Filters from 'vue2-filters';
+import <%= pageName %>Service from './<%= pageFolderName %>.service';
+
+@Component
+export default class <%= pageName %> extends mixins(Vue2Filters.mixin) {
```

Comment:
I think we can just `extends Vue` from `vue-property-decorator` and not use `Vue2Filters.mixin`